wordpress-mobile/WordPress-Aztec-iOS
# Extend title view for big titles

Closes #385
Relate to this: https://github.com/wordpress-mobile/WordPress-iOS/issues/6824

On this PR I'm converting the Title to an UITextView to allow multiple lines and adjusting the size of it depending of the text inserted.

I'm also adding a placeholder label because UITextView doesn't have placeholder labels!

How to test:
 - Start the demo app
 - On any of the demos app check the title placeholder shows correctly
 - Enter text on the title that expand to multiple lines and see if layout adapts correctly.

<img width="487" alt="screen shot 2017-03-29 at 14 07 10" src="https://cloud.githubusercontent.com/assets/651601/24455889/1ac97200-1489-11e7-809f-8efe78cdef79.png">
